ports/nrf/drivers/bluetooth/ble_drv.c

```diff
@@ -146,32 +159,55 @@ uint32_t ble_drv_stack_enable(void) {
 
     BLE_DRIVER_LOG("SoftDevice enable status: " UINT_FMT "\n", (uint16_t)err_code);
 
-#if NRF51
-    err_code = sd_nvic_EnableIRQ(SWI2_IRQn);
-#else
-    err_code = sd_nvic_EnableIRQ(SWI2_EGU2_IRQn);
-#endif
+    err_code = sd_nvic_EnableIRQ(SD_EVT_IRQn);
 
     BLE_DRIVER_LOG("IRQ enable status: " UINT_FMT "\n", (uint16_t)err_code);
-    
+
+#if (BLUETOOTH_SD == 132) && (BLE_API_VERSION == 4)
+
+    ble_cfg_t ble_conf;
+    uint32_t app_ram_start_cfg = 0x200039c0;
+    ble_conf.conn_cfg.conn_cfg_tag                     = 1;
+    ble_conf.conn_cfg.params.gap_conn_cfg.conn_count   = 1;
+    ble_conf.conn_cfg.params.gap_conn_cfg.event_length = 3;
+    err_code = sd_ble_cfg_set(BLE_CONN_CFG_GAP, &ble_conf, app_ram_start_cfg);
+
+    memset(&ble_conf, 0, sizeof(ble_conf));
+
+    ble_conf.gap_cfg.role_count_cfg.periph_role_count   = 1;
+    ble_conf.gap_cfg.role_count_cfg.central_role_count  = 1;
+    ble_conf.gap_cfg.role_count_cfg.central_sec_count   = 0;
+    err_code = sd_ble_cfg_set(BLE_GAP_CFG_ROLE_COUNT, &ble_conf, app_ram_start_cfg);
+
+    /*
+    sd_ble_cfg_set(BLE_CONN_CFG_GATTC, &ble_conf, app_ram_start_cfg);
+    sd_ble_cfg_set(BLE_CONN_CFG_GATTS, &ble_conf, app_ram_start_cfg);
+    sd_ble_cfg_set(BLE_CONN_CFG_GATT, &ble_conf, app_ram_start_cfg);
+    sd_ble_cfg_set(BLE_CONN_CFG_L2CAP, &ble_conf, app_ram_start_cfg);
+*/
```

Comment:
what is this commented-out code for?

Reply:
It was added in case i needed it. I'll remove this.